scala/scala.bzl

```diff
@@ -300,6 +278,18 @@ def scala_repositories():
       sha256 =
       "f198967436a5e7a69cfd182902adcfbcb9f2e41b349e1a5c8881a2407f615962",
   )
+  native.http_jar(
+      name = "scala_xml",
+      url =
+      "http://central.maven.org/maven2/org/scala-lang/modules/scala-xml_2.11/1.0.5/scala-xml_2.11-1.0.5.jar",
+      sha256 = "767e11f33eddcd506980f0ff213f9d553a6a21802e3be1330345f62f7ee3d50f"
+  )
+  native.http_jar(
+      name = "scala_parser_combinators",
+      url =
+      "http://central.maven.org/maven2/org/scala-lang/modules/scala-parser-combinators_2.11/1.0.4/scala-parser-combinators_2.11-1.0.4.jar",
+      sha256 = "0dfaafce29a9a245b0a9180ec2c1073d2bd8f0330f03a9f1f6a74d1bc83f62d6"
+  )
 
```

Comment:
I think we need to use `http_file` then `scala_import` them because they could in principle have macros inside, so we don't want to use ijar.